parser.js

```diff
@@ -43,6 +43,8 @@ define(function(require) {
             error = {};
             error.message = render(errorMessages[result.error.type], result.error);
             error.cursor = result.error.cursor;
+            //error.start = result.error.openTag.start;
```

Comment:
Ahh, alright I'll make sure to read this, I think I just forgot to remove that entirely though, whoops